modules/core/src/main/java/org/apache/ignite/internal/cache/query/index/sorted/keys/BytesCompareUtils.java

```diff
@@ -19,41 +19,24 @@
 
 /** */
 public final class BytesCompareUtils {
-    /** */
-    public static int compareNotNullSigned(byte[] arr0, byte[] arr1) {
```

Comment:
`BytesCompareUtils` now contains only one method. If we decided to refactor the last method, let's move it to another place like GridFunc. I think we don't need `BytesCompareUtils` any more.

Reply:
Inline method because it has single usage.